e2e/help-button.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { test, expect, type Page } from '@playwright/test';
+
+const testHelpButton = async (page: Page, videoLink: boolean) => {
+  const helpButton = page.getByTestId('get-help-dropdown');
+  await expect(helpButton).toBeVisible();
+  await helpButton.click();
+  await expect(page.getByTestId('get-hint')).toBeVisible();
+  await expect(page.getByTestId('ask-for-help')).toBeVisible();
+  if (videoLink) {
+    await expect(page.getByTestId('watch-a-video')).toBeVisible();
+  } else {
+    await expect(page.getByTestId('watch-a-video')).toBeHidden();
+  }
+};
```

Comment:
As I said in the main discussion, I don't think this approach is the way to go in this case.

DRYing out ends up making the code more verbose, since the amount of repetition we're removing is very modest.
```suggestion
```

Reply:
Yes I agree as there is a very minor amount of repetition.